apps/admin/frontend/src/components/write_in_adjudication_button.tsx

```diff
@@ -103,7 +103,10 @@ export function WriteInAdjudicationButton({
       (item) => normalizeWriteInName(item.label) === normalizedInputValue
     )
   ) {
-    options.push({ label: `Add: ${inputValue}`, value: inputValue });
+    options.push({
+      label: `Press enter to add: ${inputValue}`,
+      value: inputValue,
+    });
```

Comment:
@adghayes I added these changes after our demo on Thursday and the electioneers found them helpful. Let me know what you think (vid in the pr description)